helpme/urls.py

```diff
@@ -7,7 +7,6 @@
 urlpatterns = [
     path("", views.SupportDashboardView.as_view(), name="dashboard"),
     path("faq/", views.FAQView.as_view(), name="faq"),
-    path("request/", views.SupportRequestView.as_view(), name="submit-request"),
```

Comment:
we should bring this simple view back.  It can be very useful as a first entry point and for anonymous users submitting a ticket.